test/support/test_macros.h

```diff
@@ -13,8 +13,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef SUPPORT_TEST_MACROS_H
-#define SUPPORT_TEST_MACROS_H
```

Comment:
It seems like according to the convention of `_TEST_HEADERNAME_H`, this should be `_TEST_TEST_MACROS_H`.

Reply:
acually it should be _TEST_SUPPORT_TEST_MACROS_H

Reply:
There are no conflicts for guard name ```_TEST_TEST_MACROS_H``` now

Reply:
The convention in the main source and test source doesn't currently include the path.  If we would want to move in that direction, it would be a very large change.

There are a few other files in `test/support/test_*.h` though which probably need this same treatment.

Reply:
I think I understand what you are meaning @rarutyun, because we are prepending `_TEST_*` for these guards.  

I thought understood that convention to not mean because of the path of the guards, but more to separate the repository into the main source `_ONEDPL_*` and the test helper headers `_TEST_*` categories.  

We are creating this convention now though so we can decide what it is, as it was inconsistent before.